ticdc/ticdc-filter.md

```diff
@@ -57,31 +57,55 @@ Description of configuration parameters:
+    | Event           | Type | Alias | Description         |
+    | --------------- | ---- | -|--------------------------|
+    | all dml         |      | |Matches all DML events       |
+    | all ddl         |      | |Matches all DDL events         |
+    | insert          | DML  | |Matches `insert` DML event      |
+    | update          | DML  | |Matches `update` DML event      |
+    | delete          | DML  | |Matches `delete` DML event      |
+    | create schema   | DDL  | create database |Matches `create database` event |
+    | drop schema     | DDL  | drop database  |Matches `drop database` event |
+    | create table    | DDL  | |Matches `create table` event    |
+    | drop table      | DDL  | |Matches `drop table` event      |
+    | rename table    | DDL  | |Matches `rename table` event    |
+    | truncate table  | DDL  | |Matches `truncate table` event  |
+    | alter table     | DDL  | |Matches `alter table` event, including all clauses of `alter table`, `create index` and `drop index`   |
+    | add table partition    | DDL  | |Matches `add table partition` event     |
+    | drop table partition    | DDL  | |Matches `drop table partition` event     |
+    | truncate table partition    | DDL  | |Matches `truncate table partition` event     |
+    | create view     | DDL  | |Matches `create view`event     |
+    | drop view     | DDL  | |Matches `drop view` event     |
+    | modify schema charset and collate | DDL  | |Matches `modify schema charset and collate` event     |
+    | recover table   | DDL  | |Matches `recover table` event    |
+    | rebase auto id    | DDL  | |Matches `rebase auto id` event    |
+    | modify table comment | DDL  | |Matches `modify table comment` event    |
+    | modify table charset and collate | DDL  | |Matches `modify table charset and collate` event    |
+    | exchange table partition | DDL  | |Matches `exchange table partition` event    |
+    | reorganize table partition | DDL  | |Matches `reorganize table partition` event    |
+    | alter table partitioning | DDL  | |Matches `alter table partitioning` event    |
+    | remove table partitioning | DDL  | |Matches `remove table partitioning` event    |
+    | add column | DDL  | |Matches `add column` event    |
+    | drop column | DDL  | |Matches `drop column` event    |
+    | modify column | DDL  | |Matches `modify column` event    |
+    | set default value | DDL  | |Matches `set default value` event    |
+    | add primary key | DDL  | |Matches `add primary key` event    |
+    | drop primary key | DDL  | |Matches `drop primary key` event    |
+    | rename index | DDL  | |Matches `rename index` event    |
+    | alter index visibility | DDL  | |Matches `alter index visibility` event    |
+    | alter ttl info | DDL  | |Matches `alter ttl info` event    |
+    | alter ttl remove| DDL  | |Matches DDL events that remove all the TTL properties of a table |
```

Comment:
```suggestion
    | alter ttl remove| DDL  | |Matches DDL events that remove all TTL attributes of a table |
```

Reply:
ref: https://docs.pingcap.com/tidb/dev/time-to-live#modify-the-ttl-attribute-of-a-table

![image](https://github.com/pingcap/docs/assets/79440533/a866eece-5977-41b3-a28a-e29e58803871)